src/Robin/Ntlm/Crypt/Random/OpenSslRandomByteGenerator.php

```diff
@@ -15,7 +15,7 @@
  * "openssl" extension.
  *
  * @link http://php.net/openssl
- * @deprectated NOTE! This implementation is deprecated, as it's been found to
+ * @deprecated Insecure! This implementation is deprecated, as it's been found to
```

Comment:
> Otherwise you could make the case that every `@deprecation` tag ever warrants a `NOTE:` first. It's just a bit redundant. 

Good point. Yea. :/

> While I'm thinking about this, the deprecation tag description should probably offer some alternative, such as `... Use xyz instead.`

Another good point. I believe I linked to an issue, but you're right, usually you want to point to an implementation that should be moved to or used instead.